problems/objects.js

```diff
@@ -60,8 +77,12 @@ const appleSet = () => {};
  * @returns {number} Sum of Adam and Eve's apples.
  *
  */
-
-const adamAndEveApples = () => {};
+// let names = {"Adam": 2, "Eve": 5}
+const adamAndEveApples = (appleCountByName) => {
+  let sum = 0
```

Comment:
We don't need this sum variable. Too much work and not helping us. Just return appleCountByName.Adam + appleCountByName.Eve